lib/Ryu/Source.pm

```diff
@@ -1262,6 +1262,34 @@ sub skip_last {
     $src
 }
 
+=head2 skip_until_done
+
+Skips the items that arrive before a given Future is done.
+
+=over 4
+
+=item * A Future to wait before skip
```

Comment:
sentence does not make sense to me.   maybe  " Future  - skip until this Future is complete

Reply:
seems optional ?